source/common/http/http2/metadata_encoder.cc

```diff
@@ -127,6 +127,8 @@ std::vector<uint8_t> MetadataEncoder::payloadFrameFlagBytes() {
   return flags;
 }
 
+std::string MetadataEncoder::payload() const { return payload_.toString(); }
```

Comment:
both const method and const return value work here (i think you had added it before)